src/_includes/layouts/event.html

```diff
@@ -79,6 +79,24 @@ <h1>{{ title }}</h1>
 					</span>
 				</p>
 			</main>
+			<div id="sesh" aria-hidden="true">
```

Comment:
We should leave off `aria-hidden="true"` — `display: none;` also removes elements from the accessibility tree, but leaving `aria-hidden` on here means that even if the text is visible, screenreader users won't be able to get to it.

Reply:
great feedback! thank you.